ghcide/test/exe/WatchedFileTests.hs

```diff
@@ -28,6 +28,7 @@ tests = testGroup "watched files"
     [ testSession' "workspace files" $ \sessionDir -> do
         liftIO $ writeFile (sessionDir </> "hie.yaml") "cradle: {direct: {arguments: [\"-isrc\", \"A\", \"WatchedFilesMissingModule\"]}}"
         _doc <- createDoc "A.hs" "haskell" "{-#LANGUAGE NoImplicitPrelude #-}\nmodule A where\nimport WatchedFilesMissingModule"
+        setIgnoringRegistrationRequests False
```

Comment:
what does this do?

Reply:
It says "don't pretend that `registerCapability` and `unregisterCapability` messages don't exist", which we mostly want to otherwise it adds more noise. This test actually looks at those messages so needs to set this.